From 4fa5b3ff51eddcff5f1fa5f692d1dbea52af320a Mon Sep 17 00:00:00 2001
From: Julien Olivain <julien.olivain@nxp.com>
Date: Wed, 16 Nov 2016 09:39:46 +0100
Subject: [PATCH 2194/5242] MLK-13473: imx_sim: fix module device table name

commit  d9f2e6641d895ad25f7d52211c98d9170a9dfb9f from
https://source.codeaurora.org/external/imx/linux-imx.git

This fixes the build when this driver is built as a module, when
CONFIG_MXC_SIM=m

Signed-off-by: Julien Olivain <julien.olivain@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/sim/imx_sim.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/sim/imx_sim.c b/drivers/mxc/sim/imx_sim.c
index 439561a..14dd12a 100755
--- a/drivers/mxc/sim/imx_sim.c
+++ b/drivers/mxc/sim/imx_sim.c
@@ -1709,7 +1709,7 @@ enum imx_sim_type {
 	{ /* sentinel */ }
 };
 
-MODULE_DEVICE_TABLE(of, mlb150_imx_dt_ids);
+MODULE_DEVICE_TABLE(of, sim_imx_dt_ids);
 
 static int sim_probe(struct platform_device *pdev)
 {
-- 
1.7.9.5

